### PR TITLE
Remove multiple instances of Layer5 and fix indentation of bulleted list

### PR DESCRIPTION
**Pull request by Aritra8438**
**Notes for Reviewers**
This PR addresses some minor documentation modifications.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit a1063de71c73e9a2eb1efdba63745d9d88bdad15 at: https://64cb84af9088c8630c01adcc--meshery-docs-preview.netlify.app

File: docs/pages/project/community.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ The Meshery project is community-built and welcomes collaboration!
 <br /><a href="https://github.com/meshery/meshery"><i class="fab fa-github"></i> Fork here on Github</a>
 
 * Join any of the [community meetings](https://meshery.io/calendar). 
-  * Watch [meeting recordings](https://www.youtube.com/@Layer5io?sub_confirmation=1) and subscribe to the [community calendar](https://meshery.io/calendar).
+* Watch [meeting recordings](https://www.youtube.com/@Layer5io?sub_confirmation=1) and subscribe to the [community calendar](https://meshery.io/calendar).
```
Comment (leecalcote) [inline docs/pages/project/community.md:21]: this indentation is intentional. please revert.

**Review by leecalcote — CHANGES_REQUESTED** [review]
Thanks for your PR, @Aritra8438.

File: docs/pages/project/community.md (hunk 1 above)
Comment (Aritra8438) [reply docs/pages/project/community.md]: Done, @leecalcote, PTAL, Thanks!

**Comment by l5io** [comment]
🚀 Preview for commit 4ad9e87dd43740cf4ea341da8feb114ac77ab41b at: https://64cbaa2e940fb97b1a89b3a6--meshery-docs-preview.netlify.app

**Review by leecalcote — APPROVED** [review]
Looks good.

**Pull request merged**